docs/configuration/smime_email_signing.rst

```diff
@@ -0,0 +1,302 @@
+
+.. note::
+    S/MIME signing is currently only available when using the SMTP Email transport. It doesn't work with API-based Email transports.
+
+
+
+What is S/MIME email signing?
+*****************************
+
+S/MIME Email signing adds a digital signature to your Emails, which allows recipients to:
+
+1. **Verify the sender's identity** - Confirm that the Email actually came from your organization
+2. **Ensure Email integrity** - Verify that the Email content wasn't tampered with during transmission
+3. **Build trust** - Demonstrate to recipients that your organization follows Email security best practices
+
+.. note::
+    This implementation focuses on **signing** Emails only. It doesn't encrypt the Email body, which means the Email content is still readable to anyone who has access to it. Many Email clients don't support S/MIME encryption, so signing provides verification without compatibility issues.
+
+For more detailed information about S/MIME and why to use it, see these articles from SparkPost:
+
+- :xref:`SparkPost S/MIME Overview`
+- :xref:`SparkPost S/MIME Technical Guide`
+
+How it works
+************
+
+When you turn on and properly configure S/MIME signing:
+
+1. Mautic signs each outgoing Email with the private key for the sender's Email address
+2. Mautic attaches the signature to the Email as a ``smime.p7s`` file
+3. Recipients with S/MIME-capable Email clients can verify the signature using the public certificate
+4. If no certificate exists for a sender's Email address, Mautic sends the Email unsigned without error
+
+Turn on S/MIME signing
+**********************
+
+Turn on S/MIME signing through your Mautic configuration file. It isn't available in the web interface.
+
+.. warning::
+    Make sure you're using the SMTP Email transport before turning on S/MIME signing. The feature doesn't work with API-based Email transports.
+
+Configuration
+=============
+
+Add the following configuration parameters to your ``app/config/local.php`` file:
+
+.. code-block:: php
+
+    <?php
+    $parameters = array(
+        // ... other configuration ...
+        'smime_signing_enabled' => true,
+        'smime_certificates_path' => '%kernel.project_dir%/var/smime_certificates',
+    );
+
+Configuration parameters
+------------------------
+
+``smime_signing_enabled``
+    set to ``true`` to turn on S/MIME Email signing or ``false`` to turn it off.
+    
+    Default: ``false``
+
+``smime_certificates_path``
+    the absolute path to the directory where Mautic stores your S/MIME certificates. You can use ``%kernel.project_dir%`` to reference your Mautic installation directory.
+    
+    Default: ``%kernel.project_dir%/var/smime_certificates``
+
+Generating S/MIME certificates
+*******************************
+
+Each Email address that sends Emails from Mautic needs its own pair of certificates:
+
+- A **public certificate** (``.crt`` file) that verifies your identity
+- A **private key** (``.pem`` file) that signs the Emails
```

Comment:
Maybe we can use dash here instead of parantheses.


```suggestion
- A **public certificate** - ``.crt`` file - that verifies your identity
- A **private key** - ``.pem`` file - that signs the Emails
```